tests/posix/common/src/pthread.c

```diff
@@ -233,6 +233,33 @@ void *thread_top_term(void *p1)
 	return NULL;
 }
 
+ZTEST(posix_apis, test_pthread_condattr)
+{
+	int ret;
+	clockid_t clock_id;
+	pthread_condattr_t att;
+
+	ret = pthread_condattr_init(&att);
+	zassert_equal(ret, 0, "pthread_condattr_init failed");
+
+	ret = pthread_condattr_getclock(&att, &clock_id);
+	zassert_equal(ret, 0, "pthread_condattr_getclock failed");
```

Comment:
Would be good to apply this pattern elsewhere
```suggestion
	zassert_ok(pthread_condattr_getclock(&att, &clock_id));
```

Reply:
Done, replaced `zassert_equal` with `zassert_ok` wherever i could and removed `ret` variable